models/optional_modules/conversions/scratch/snowplow_unified_conversions_this_run.sql

```diff
@@ -13,35 +13,15 @@ You may obtain a copy of the Snowplow Personal and Academic License Version 1.0
 }}
 
 with prep as (
-
-{%- for conv_def in var('snowplow__conversion_events', []) %}
-
-  select
-    
-    session_identifier,
-    user_identifier,
-    {% if var('snowplow__conversion_stitching') %}
-    -- updated with mapping as part of post hook on derived conversions table
-    cast(user_identifier as {{ snowplow_utils.type_max_string() }}) as stitched_user_id,
-    {% endif %}
-    cv_{{ conv_def['name'] }}_volume as cv_volume,
-    cv_{{ conv_def['name'] }}_values as cv_values,
-    cv_{{ conv_def['name'] }}_total as cv_total,
-    cv_{{ conv_def['name'] }}_events as cv_events,
-    cv_{{ conv_def['name'] }}_first_conversion as cv_first_conversion,
-    cv_{{ conv_def['name'] }}_converted as cv_converted,
-    '{{ conv_def['name'] }}' as cv_type
   
-  from {{ ref('snowplow_unified_sessions_this_run') }} as ev
-  
-  where cv_{{ conv_def['name'] }}_converted
+ {%- for conv_def in var('snowplow__conversion_events') %}
+ 
+    {{ snowplow_unified.get_conversion_fields(conv_def)}}
+    {% if not loop.last %}union all{% endif %}
```

Comment:
Might be best to add it as an argument to the macro, but we should support passthrough fields in this table as well :)